goss-testing/dat/goss-servers.cfg

```diff
@@ -35,3 +35,7 @@
 9003       ncn-afterpitreboot-kubernetes-tests-worker-single.yaml  worker
 
 9004       ncn-storage-tests.yaml                                  storage
+
+9005       ncn-cms-tests.yaml                                      master
```

Comment:
Oh, it's not a big deal, but it is slightly better if you run the CMS tests on a worker node. That allows one additional TFTP check to run.

Reply:
I've added csm test suite startup to worker node. On test invocation framework side, I'll invoke cms tests on worker.

Reply:
Re-ran test framework with CMS tests mved from master to worker: https://jenkins.algol60.net/blue/organizations/jenkins/Cray-HPE%2Fcsm-vshasta-deploy/detail/feature%2Fhms-cms-tests/40/tests/
I don't see any difference from previous runs, where CMS tests were invoked on master, hence I'm considering this change successfully tested.